examples/cloud-router-2-aws-connection/README.md

```diff
@@ -289,6 +289,279 @@ resource "aws_dx_private_virtual_interface" "aws_virtual_interface" {
 ```
 <!-- End Example Usage -->
```

Comment:
Need to remove all `<!-- Begin Example Usage --><!-- End Example Usage -->` sections (including their contents). You're replacing them and we need to see the full diff. It should be done in every example README.md file.

Reply:
Addressed with [commit](https://github.com/equinix/terraform-equinix-fabric/pull/102/commits/4e7ee85646d9482c993afb4565814c3b383a556b)